src/chrome/content/rules/bbk.bund.de.xml

```diff
@@ -1,4 +1,11 @@
 <!--
+	For other bund.de coverages, see:
+ 	- Federal-Office-for-Information-Security.xml
+	- Bundesnachrichtendienst.xml
+	- Bundesministerium_des_Innern.xml
+	- Federal_Office_of_Administration.xml
+	- Verwaltung_Online.xml
```

Comment:
We use to make a reference from the referenced files back to this one. See examples in `Google.xml` or `Amazon.xml`.